extensions/redis-cache/runtime/src/main/java/io/quarkus/cache/redis/runtime/RedisCacheBuildTimeConfig.java

```diff
@@ -3,20 +3,17 @@
 import java.util.Optional;
 
 import io.quarkus.runtime.annotations.ConfigGroup;
-import io.quarkus.runtime.annotations.ConfigItem;
 
 @ConfigGroup
```

Comment:
Do we still need this annotation? I thought we only need `@ConfigMapping` (and sometimes `@ConfigRoot(phase)`) on the root config interface and that's all?

Reply:
We don't strictly need it. But it doesn't hurt and I use it for documentation purposes more than anything.

Reply:
OK, understood.